io-engine/src/subsys/config/opts.rs

```diff
@@ -668,7 +668,7 @@ impl Default for IoBufOpts {
     fn default() -> Self {
         Self {
             small_pool_count: try_from_env("IOBUF_SMALL_POOL_COUNT", 8192),
-            large_pool_count: try_from_env("IOBUF_LARGE_POOL_COUNT", 1024),
+            large_pool_count: try_from_env("IOBUF_LARGE_POOL_COUNT", 2048),
```

Comment:
This config might be performance sensitive. I'd recommend doing a sanity perf run with and without this change. Also, sometime back I was reading spdk issue #809, where it says 
> It’s always best to specify a value for –n that is one less than a power of two (2^x -1) because of the way rings work in DPDK. You end up allocating a ring twice the size for the one extra element between 511 and 512 for example.

So along with this, we can consider changing NVMF_TCP_NUM_SHARED_BUF from 2048 to 2047 as well.

Reply:
Changed, and for my limited testing I see no perf impact.